src/develop/imageop.c

```diff
@@ -1968,8 +1968,10 @@ gboolean _iop_validate_params(dt_introspection_field_t *field,
     }
     break;
   case DT_INTROSPECTION_TYPE_FLOAT:
-    all_ok = dt_isnan(*(float*)p)
-      || ((*(float*)p >= field->Float.Min && *(float*)p <= field->Float.Max));
+     all_ok = dt_isnan(*(float*)p)
+      || dt_isinf(*(float*)p)
+      || (*(float*)p == field->Float.Default)
```

Comment:
I've read your comment but I do not see a default outside of MIN .. MAX range. Is there really a case on the dt params? If so we may want to fix the introspection maybe, I have hard time to understand how a default could be set outside the MIN .. MAX range.

I'm ok for `isinf`.

Reply:
For one of the modules using aspect ratios (probably framing, but could be crop), the introspection range had to be increased to permit special flag values like "original" or "freehand".

Reply:
Sure for border

Reply:
Got it, thanks!